docs/apache-airflow/upgrading-to-2.rst

```diff
@@ -508,6 +508,9 @@ At this point, just follow the standard Airflow version upgrade process:
 
       The database upgrade may take a while depending on the number of DAGs in the database and the volume of history
       stored in the database for task history, xcom variables, etc.
+      In our testing, we saw that performing the Airflow database upgrade from Airflow 1.10.14 to Airflow 2.0
+      took between two to three minutes on an Airflow database on PostgreSQL with around 35,000 task instances and
+      500 DAGs.
```

Comment:
Should we add a note like "However, this time may depend on your infrastructure and network latencies"? Just to avoid benchmarking. But that's just idea